great_expectations/expectations/core/expect_column_max_to_be_between.py

```diff
@@ -129,6 +129,8 @@ def validate_configuration(self, configuration: Optional[ExpectationConfiguratio
         super().validate_configuration(configuration)
         self.validate_metric_value_between_configuration(configuration=configuration)
 
+        return True
+
     @classmethod
```

Comment:
Would you mind type annotating the return value of all these Expectations? Thanks so much for fixing this 🙇🏽

Reply:
For all Expectations or just the ones that were modified in this branch?

Reply:
Just the ones modified in this branch! I think we can incrementally refactor and improve our code but I don't want to increase the scope of your work here.

As we touch the other files, we can update those accordingly.

Reply:
Too late, haha.